components/compat-table/utils.js

```diff
@@ -2,7 +2,29 @@
  * A list of browsers to be hidden.
  * @constant {string[]}
  */
-export const HIDDEN_BROWSERS = ["bun", "ie"];
+export const SHOW_BROWSERS = [
+  "chrome",
+  "edge",
+  "firefox",
+  "opera",
+  "safari",
+  "chrome_android",
+  "firefox_android",
+  "opera_android",
+  "safari_ios",
+  "samsunginternet_android",
+  "webview_android",
+  "webview_ios",
+  "bun",
+  "deno",
+  "nodejs",
+];
+
+/**
+ * A list of browsers to be hidden.
+ * @constant {string[]}
+ */
+export const HIDDEN_BROWSERS = ["ie"];
```

Comment:
Since this isn't used anywhere, shall we remove? It'll be confusing to maintain an allowlist and a blocklist.